src/author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
+  attr_accessor :items
+  attr_reader :first_name, :last_name
+
+  def initialize(id, first_name, last_name)
```

Comment:
Nice job so far brother. 

As mentioned in the earlier comment, i would strongly suggest you use auto-generated id in this class initilizer so that users doesn't need to explicitly put in an id as id of the same number might be inserted twice and does corrupting or dataset. 

the line of code below, would be helpful and goes inline with my suggestion. 

`def initialize(first_name, last_name, id: SecureRandom.hex(7))`
    `    @id = id`
    `    @first_name = first_name`
    `    @last_name = last_name`
    `    @items = []`
  `end`

Reply:
Ohh....what a helpful comment! Thank you so much, brother. your comments are really nice and clear. I will walk through it.